From cad2dbc4e58fb1df8918fd0cf0f93c0d36fcfec3 Mon Sep 17 00:00:00 2001
From: Fugang Duan <b38611@freescale.com>
Date: Wed, 10 Dec 2014 18:55:04 +0800
Subject: [PATCH 0327/1594] MLK-10199 ARM: clk-imx6q: set enet pll rate to
 125Mhz

commit 7a1003e6d4aa8f6d5271b2891b1727b63663d4a2 from
git://git.freescale.com/imx/linux-2.6-imx.git

Set enet pll rate to 125Mhz for RGMII tx refrence clock to
support i.MX6q sabreauto cpu2 board.

changed the enet_ref clock name for 3.14 kernel.

Signed-off-by: Fugang Duan <B38611@freescale.com>
(cherry picked from commit 6ee3d49271b1f13e359c1acde189dbb6dc4cb13e)
---
 arch/arm/mach-imx/clk-imx6q.c |    3 +++
 1 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/arch/arm/mach-imx/clk-imx6q.c b/arch/arm/mach-imx/clk-imx6q.c
index dfb659d..ce99a64 100644
--- a/arch/arm/mach-imx/clk-imx6q.c
+++ b/arch/arm/mach-imx/clk-imx6q.c
@@ -574,6 +574,9 @@ static void __init imx6q_clocks_init(struct device_node *ccm_node)
 		imx_clk_prepare_enable(clk[IMX6QDL_CLK_USBPHY2_GATE]);
 	}
 
+	/*Set enet_ref clock to 125M to supply for RGMII tx_clk */
+	clk_set_rate(clk[IMX6QDL_CLK_ENET_REF], 125000000);
+
 #ifdef CONFIG_MX6_VPU_352M
 	/*
 	 * If VPU 352M is enabled, then PLL2_PDF2 need to be
-- 
1.7.5.4

